packages/jsii-dotnet-generator/Directory.Build.props.t.js

```diff
@@ -1,10 +1,9 @@
-const path = require('path');
-const version = require('./package.json').version.replace(/\+.+$/, ''); // omit "+build" postfix;
+const version = require('./package.json').version.replace(/\+.+$/, ''); // omit "+build" suffix
```

Comment:
Why do we need to publish the code generator to NuGet? Isn't it just bundled as part of `jsii-pacmak`? I don't see a use case for someone to use it directly...

Reply:
It isn't necessary, I just included it for completeness. Will remove.